workbench-core/accounts/src/services/costCenterService.ts

```diff
@@ -44,7 +44,7 @@ export default class CostCenterService {
    */
   public async softDeleteCostCenter(
     request: DeleteCostCenterRequest,
-    checkDependency: (costCenterId: string) => void
+    checkDependency: (costCenterId: string) => Promise<void>
```

Comment:
The [function](https://github.com/aws-solutions/solution-spark-on-aws/blob/develop/solutions/swb-app/src/costCenterRoutes.ts#L45) we're passing in is a `Promise` so the parameter type should be a `Promise` as well.

Reply:
Why compiler didn't complain?